src/junglescout/models/requests/keyword_by_asin_request.py

```diff
@@ -64,6 +64,4 @@ def build_params(self, params: KeywordByAsinParams) -> Dict:  # noqa: PLR6301
         return params.model_dump(by_alias=True, exclude_none=True)
 
     def build_payload(self, attributes: KeywordByAsinAttributes) -> str:
-        return json.dumps(
-            {"data": {"type": self.type.value, "attributes": attributes.model_dump(by_alias=True, exclude_none=True)}}
-        )
+        return {"data": {"type": self.type.value, "attributes": attributes.model_dump(by_alias=True, exclude_none=True)}}
```

Comment:
Can you clarify why we need to return a `dict` here, rather than a JSON formatted string?

Reply:
For httpx the json should a JSON serializable object, not the serialized object.

Reply:
Makes sense!